tavern/internal/namegen/namegen.go

```diff
@@ -886,21 +888,41 @@ var (
 	}
 )
 
-// GetRandomName generates a random name from the list of adjectives and surnames in this package
-// formatted as "adjective-surname". For example 'focused-turing'.
-func GetRandomName() string {
+type Complexity int
+
+const (
+	Simple   Complexity = iota // 0
+	Moderate                   // 1
+	Complex                    // 2
+)
+
+// GetRandomName generates a random name based on the complexity.
+func GetRandomName(complexity Complexity) string {
+	var adj1, adj2, noun string
+	var num int64
+
 	if time.Now().Month() == time.October && time.Now().Day() == 31 {
```

Comment:
Ik outside the scope of the PR but if you have revisions could you remove the day == 31 so the whole month of Oct is spooky names.